docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
 $ docker pull container-registry.oracle.com/middleware/soasuite:12.2.1.3
 ```
 
-Additional information about using this image is available in the
-[Oracle Container Registry](https://container-registry.oracle.com).
-
 #### Creating a SOA Suite Docker image
 
-If you prefer, you can also create a Docker image containing the SOA Suite binaries yourself.
-A [sample](https://github.com/oracle/docker-images/tree/master/OracleSOASuite)
-is provided in the Oracle GitHub account that demonstrates how to create a Docker image
-to run SOA Suite.  
+You can also create a Docker image containing the Oracle SOA Suite binaries. This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches.
 
-Please consult the [README](https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/README.md) file associated with this sample for important prerequisite steps,
-such as building or pulling the Server JRE Docker image, Oracle FMW Infrastructure Docker image, and downloading the SOA Suite installer binary.
+Please consult the [README](https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/README.md) file for important prerequisite steps,
+such as building or pulling the Server JRE Docker image, Oracle FMW Infrastructure Docker image, and downloading the Oracle SOA Suite installer and bundle patch binaries.
 
-You must also install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
-to use this image with the operator.  A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s)
-is provided that demonstrates how to create a Docker image with the necessary patch installed. Use this patched image for building the SOA Suite image.
+For the Fusion Middleware Infrastructure image you must install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
```

Comment:
For the Fusion Middleware Infrastructure image you must -> For the Fusion Middleware Infrastructure image, you must
[required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}}). (Put a period at the end of the sentence.)